new-frontiers-14/frontier-station-14
# Fix Frontier Outpost atmos issues

## About the PR
Rework the piping on Frontier Outpost so that the publicly accessible connector ports are completely disconnected from the station's distro.

## Why / Balance
In a recent round on Maunder, someone filled *so many canisters* that Frontier Outpost's atmosphere became saturated with nitrogen. Frontier Outpost literally ran out of oxygen. By separating the public connector ports from the normal distro, we can prevent this from being a problem in the future.

## How to test
1. Remove the canisters that are normally present on Frontier Outpost.
2. Spawn (empty) storage canisters in their place.
3. Wrench them, let them fill up.
4. Repeat step 2 a bunch of times.
5. Use a gas analyzer to ensure that Frontier's air vents are outputting 79% nitrogen and 21% oxygen.
6. Also ensure that the storage canisters fill up with the right gases, particularly on the air ports.

## Media
![image](https://github.com/user-attachments/assets/18b4858e-5fa1-4b5d-ae7a-f69c6d6f8cb5)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

## Breaking changes
No.

**Changelog**
:cl:
- fix: Frontier Outpost won't run out of air if you fill up the canisters outside engineering.
